randovania/games/prime1/generator/bootstrap.py

```diff
@@ -71,6 +72,21 @@ def prime1_progressive_damage_reduction(self, db: ResourceDatabase, current_reso
 
         return dr
 
+    def prime1_additive_damage_reduction(self, db: ResourceDatabase, current_resources: ResourceCollection) -> float:
```

Comment:
Add a test for this function in https://github.com/randovania/randovania/blob/b2eefaaf0be3ee539a2205514e8c7a80c277aa2a/test/games/prime1/generator/test_prime_bootstrap.py, to match how the other 2 have tests already